skops/io/_persist.py

```diff
@@ -28,6 +28,29 @@
 
 
 def save(obj, file):
+    """Save an object using the skops persistence format.
+
+    Skops aims at providing a secure persistence feature that does not rely on
+    :mod:`pickle`, which is inherently insecure. For more information, please
+    visit the :ref:`persistence` documentation.
+
+    .. warning::
+
+       This feature is very early in development, which means the API is
+       unstable and it is **not secure** at the moment. Therefore, use the same
+       caution as you would for ``pickle``: Don't load from sources that you
+       don't trust. In the future, more security will be added.
+
+    Parameters
+    ----------
+    obj
+        The object to be saved
+
+    file: str
+        The file name. A zip archive will automatically created, with the ".zip"
+        ending added to the given filename.
```

Comment:
we're adding a `skops` extention, not a `zip` one.

Reply:
Done.